eiffel-vocabulary/EiffelSourceChangeCreatedEvent.md

```diff
@@ -0,0 +1,210 @@
+# EiffelSourceChangeCreatedEvent
+The EiffelSourceChangeCreatedEvent declares that a change to sources has been made, but not yet submitted (see [EiffelSourceChangeSubmittedEvent](./EiffelSourceChangeSubmittedEvent.md)). This can be used to represent a change done on a private branch, undergoing review or made in a forked repository. Unlike EiffelSourceChangeSubmittedEvent, EiffelSourceChangeCreatedEvent _describes the change_ in terms of who authored it and which issues it addressed.
+
+Where changes are integrated (or "submitted") directly on a shared branch of interest (e.g. "master", "dev" or "mainline") both EiffelSourceChangeCreatedEvent and EiffelSourceChangeSubmittedEvent are sent together.
+
+## Data Members
+### data.author
+__Type:__ Object  
+__Required:__ No  
+__Description:__ The author of the change.
+
+#### data.author.name
+__Type:__ String  
+__Required:__ No  
+__Description:__ The name of the author.
+
+#### data.author.email
+__Type:__ String  
+__Required:__ No  
+__Description:__ The email address of the author.
+
+#### data.author.id
+__Type:__ String  
```

Comment:
I think it's fine to include non-mandatory fields which may or may not be relevant in each particular case, the principle being "You're free to decide what to say, but if you want to say it, here's how".